src/bin/bench.rs

```diff
@@ -185,19 +188,40 @@ fn perform_tree_operations(tree: Tree, config: Config) {
         (Op::Delete, config.delete),
         (Op::Cas, config.cas),
     ];
-
     // TODO update to handle division errors
     let ops_per_thread = config.num_operations / config.num_threads;
     let mut threads = Vec::new();
+    let statistics = Statistics::new();
+    let (tx, rx) = channel();
+
+    let statistics = Arc::new(statistics);
+    thread::spawn(move || {
+        loop {
+            // TODO how do I get the most recent value here rather than receiving one at a time?
+            let ops_per_second = rx.recv().unwrap();
```

Comment:
rather than using a channel, you can just have other threads do `fetch_add` on the counter to increment it, and then this thread just calls `swap(0)` on the counter once per second, which resets the value to 0 and returns the previous count.